scripts/build.sh

```diff
@@ -1,51 +1,51 @@
 #!/usr/bin/env bash
-echo "Run with 'source ./scripts/build.sh [fed_size] [dir]"
+echo "Run with 'source ./scripts/build.sh [FM_FED_SIZE] [dir]"
```

Comment:
`dir` -> `FM_TMP_DIR` for consistency

Reply:
Might be better to call it `FM_DIR` because it isn't necessarily a temporary dir. Could be `./cfg` for instance.

Reply:
Ah, that was an oversight, will revert. I let `sed` do the heavy lifting and then went through the docs and fixed the places where it shouldn't be changed. Didn't notice the help text there.